nixos/tests/graphite.nix

```diff
@@ -31,16 +35,17 @@ import ./make-test-python.nix ({ pkgs, ... } :
     one.wait_for_unit("default.target")
     one.wait_for_unit("graphiteWeb.service")
     one.wait_for_unit("graphiteApi.service")
-    one.wait_for_unit("graphitePager.service")
     one.wait_for_unit("graphite-beacon.service")
     one.wait_for_unit("carbonCache.service")
-    one.wait_for_unit("seyren.service")
+    # one.wait_for_unit("seyren.service")
```

Comment:
Remove line?

Reply:
seyren integration is not broken, it « just » indirectly depends on a un-secure version of openssl. I kept the line to make it easier to re-enable it, but I have no problem dropping this line.

Reply:
done.